lib/parse-message.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+//socket = one user, s (when in a loop) = all users
```

Comment:
Commenting for yourself is great so you can understand what's going on. For the future, try to do this in your own branch, and then submit a separate branch that has no comments or console.logs, just to get in the best practice of getting rid of stuff before a real deployment.